vaadin-testbench-unit/src/main/java/com/vaadin/flow/component/checkbox/CheckboxGroupWrap.java

```diff
@@ -0,0 +1,189 @@
+/*
+ * Copyright (C) 2022 Vaadin Ltd
+ *
+ * This program is available under Commercial Vaadin Developer License
+ * 4.0 (CVDLv4).
+ *
+ *
+ * For the full License, see <https://vaadin.com/license/cvdl-4.0>.
+ */
+
+package com.vaadin.flow.component.checkbox;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.function.BiConsumer;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.jetbrains.annotations.NotNull;
+
+import com.vaadin.flow.data.binder.HasItemComponents;
+import com.vaadin.testbench.unit.ComponentWrap;
+import com.vaadin.testbench.unit.Wraps;
+
+/**
+ * Test wrapper for CheckboxGroup components.
+ *
+ * @param <T>
+ *            component type
+ */
+@Wraps(fqn = "com.vaadin.flow.component.checkbox.CheckboxGroup")
+public class CheckboxGroupWrap<T extends CheckboxGroup<V>, V>
+        extends ComponentWrap<T> {
+
+    /**
+     * Wrap given component for testing.
+     *
+     * @param component
+     *            target component
+     */
+    public CheckboxGroupWrap(T component) {
+        super(component);
+    }
+
+    @Override
+    public boolean isUsable() {
+        return super.isUsable() && !getComponent().isReadOnly();
+    }
+
+    /**
+     * Select items by client string representation.
+     *
+     * @param selection
+     *            item string representation
+     * @param others
+     *            other items string representation
+     */
+    public void selectItem(String selection, String... others) {
+        ensureComponentIsUsable();
+        selectItem(toCollection(selection, others));
+    }
+
+    /**
+     * Select items by client string representation.
+     *
+     * @param selection
+     *            items string representation
+     */
+    public void selectItem(Collection<String> selection) {
```

Comment:
```suggestion
    public void selectItems(Collection<String> selection) {
```
Items as it is targeting a collection

Reply:
done